source/credit/index.rst.old

```diff
@@ -0,0 +1,58 @@
+Credit
```

Comment:
Should not have been added

Reply:
Sorry,  it's a mistake. i passed the PR in review too early

Reply:
This article is ok for review